docker-compose.client.yml

```diff
@@ -11,3 +11,7 @@ services:
         stdin_open: true
         tty: true
         privileged: true
+        tmpfs:
+            - /data
+            - /tmp
+            - /mnt
```

Comment:
I am curious: why do we need these? Does it considerably improve the client performance?